### DISCUSSION TITLE
Jupyter notebook file rapidly flickering, when opening in github

### DISCUSSION THREAD

**Discussion by peterw-github** [community/community#48325] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hey guys!

Unfortunately when I open a particular juypter notebook file, it starts rapidly flickering beyond a certain point. This is what the flickering looks like:

https://gifyu.com/image/Sqe98

I've opened other jupyter notebook files in my other github repos, and they were fine. I've also tried opening this jupyter notebook file in other browsers, and I get the same issue. Opening the notebook using actual Jupyter Notebook has no issues. 

Any help would be appreciated, I'm quite stumped.

Edit: Interestingly in the GIF, the flickering is much slower, when actually viewing it in github, the flickering looks to be about 5 times faster or so.

**Comment by sbhask** [comment, marked as answer]
Could this ben an issue with the level of zoom you have in your browser?

**Reply by peterw-github** [reply]
Ah that's fixed it, zooming to >=110%, or  <=90% works perfectly, cheers Sam!
